lightning-block-sync/src/init.rs

```diff
@@ -239,8 +239,6 @@ impl<'a, L: chain::Listen + ?Sized> chain::Listen for DynamicChainListener<'a, L
 struct ChainListenerSet<'a, L: chain::Listen + ?Sized>(Vec<(u32, &'a L)>);
 
 impl<'a, L: chain::Listen + ?Sized> chain::Listen for ChainListenerSet<'a, L> {
-	// Needed to differentiate test expectations.
-	#[cfg(test)]
```

Comment:
It looks like it was calling each listener's `filtered_block_connected` method, though, instead of `block_connected`. Is that a problem? Not clear from the commit message why.